relay-dynamic-config/src/defaults.rs

```diff
@@ -176,6 +176,9 @@ fn span_metrics() -> impl IntoIterator<Item = MetricSpec> {
                 Tag::with_key("span.main_thread")
                     .from_field("span.sentry_tags.main_thread")
                     .when(is_mobile.clone()),
+                Tag::with_key("span.app_start_type")
```

Comment:
Please add the tag key to https://github.com/getsentry/sentry/blob/master/src/sentry/sentry_metrics/indexer/strings.py before merging this PR, to prevent creating a copy of this tag key in the indexer db for every org.

Reply:
PR is up for adding it here: https://github.com/getsentry/sentry/pull/64356